test/integration/policy_info_metric_test.go

```diff
@@ -18,6 +19,9 @@ const (
 	propagatorMetricsSelector = "component=ocm-policy-propagator"
 	ocmNS                     = "open-cluster-management"
 	metricName                = "policy_governance_info"
+	saName					  = "grc-framework-sa"
+	saNamespace				  = "kube-system"
```

Comment:
We shouldn't be messing with the `kube-system` namespace in our tests. Is there another namespace (new or one created for our tests) that we could use? Maybe just reuse `userNamespace`?